examples/material_gallery/lib/demo/page_selector_demo.dart

```diff
@@ -2,49 +2,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:flutter/animation.dart';
 import 'package:flutter/material.dart';
 
 class PageSelectorDemo extends StatelessComponent {
-  Widget _buildTabIndicator(BuildContext context, String iconName) {
```

Comment:
The reason I did it in this patch is that I did the move in order to add the accessibility logic (it requires refactoring where the animation happens so that I have the builder outside all the dots instead of per-dot, and once I was rewriting the whole thing anyway, I figured I'd move it at the same time). But I can do the move without the accessibility logic, then just patch it in this patch if you prefer.